UKHomeOffice/sgmr-service
# SPCR-534 Cancel original voyage if it is being 'updated'

### AC / Description of changes

- include a previousVoyageReportId on the new voyage when a submitted voyage is changed (change creates a newVoyageId and PATCHes to it)
- when updated voyage is submitted, cancel the previous voyage


### To Test

- create a new voyage
- submit it
- make a note of it
- change something on the voyage
- submit the changed voyage
- > you should see the original voyage in 'cancelled'
- > you should see the new (updated) voyage in 'submitted
- > you should receive a cancellation email
- > you should receive a new submitted email



### Notes

---

**Remember to merge to feature branches not `master`.**
